src/main/java/dk/bec/bookanything/service/UserService.java

```diff
@@ -0,0 +1,44 @@
+package dk.bec.bookanything.service;
+
+import dk.bec.bookanything.model.UserEntity;
+import dk.bec.bookanything.repository.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.web.context.annotation.ApplicationScope;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+@ApplicationScope
+public class UserService {
+
+    private final UserRepository userRepository;
+
+    @Autowired
+    public UserService(UserRepository userRepository){
+        this.userRepository = userRepository;
+    }
+
+    public UserEntity addUser(UserEntity userEntity){
+        return userRepository.save(userEntity);
+    }
+
+    public List<UserEntity> getUsers(){
+        return userRepository.findAll();
+    }
+
+    public Optional<UserEntity> getUserById(Long id){
+        return userRepository.findById(id);
+    }
+
+    public void deleteUserById(Long id){
+        Optional<UserEntity> userEntityOptional = getUserById(id);
+        userEntityOptional.ifPresent(userRepository::delete);
```

Comment:
When user is not present it could be logger with warn or debug log (also applies to delete operation)